pkg/security/secl/compiler/ast/secl.go

```diff
@@ -158,7 +158,7 @@ type ScalarComparison struct {
 type ArrayComparison struct {
 	Pos lexer.Position
 
-	Op    *string `parser:"( @( \"in\" | \"not\" \"in\" | \"intersects\" )"`
+	Op    *string `parser:"( @( \"in\" | \"not\" \"in\" | \"allin\" )"`
```

Comment:
Do you think we should implement "allin" for something else than CIDR ? (like [ "XXX", "YYY" ] allin process.capabilities ?

Reply:
Probably, let's add the in a following PR